staging/src/k8s.io/apiserver/pkg/server/config_test.go

```diff
@@ -283,11 +283,6 @@ func TestAuthenticationAuditAnnotationsDefaultChain(t *testing.T) {
 		// confirm that we can set an audit annotation in a handler before WithAudit
 		audit.AddAuditAnnotation(req.Context(), "pandas", "are awesome")
 
-		// confirm that trying to use the audit event directly would never work
```

Comment:
maybe we should confirm here that the level is set to `LevelNone`?

Reply:
(See above comments)